.github/workflows/commands.yml

```diff
@@ -0,0 +1,22 @@
+name: Repo Commands
+
+on:
+  issue_comment:        # Handle comment commands
+    types: [created]
+  pull_request:         # Handle renamed PRs
+    types: [edited]
+
+jobs:
+  comment-trigger:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v3
+    - name: Run command handlers
+      uses: PrismarineJS/prismarine-repo-actions@master
+      with:
+        # NOTE: You must specify a Personal Access Token (PAT) with repo access here. While you can use the default GITHUB_TOKEN, actions taken with it will not trigger other actions, so if you have a CI workflow, commits created by this action will not trigger it.
+        token: ${{ secrets.PAT_PASSWORD }}
+        # See `Options` section below for more info on these options
+        install-command: npm install
+        /fixlint.fix-command: npm run fix
```

Comment:
are you sure this will work here?

mcdata is a bit different from all other repos

there's tools/js for package.json stuff

and for releases there's only HISTORY.md , no package.json

Reply:
Yeah, the package.json is optional, it can read the version from just the history https://github.com/extremeheat/minecraft-data/pull/17

Reply:
how does it know where to run npm run fix ?

Reply:
It runs the commands specified in the workflow options at the root of the repo. But yeah it was missing the “cd tools/js” step, fixed that in the workflow options